### PR TITLE
Sprint 45: tlt 2501 -- help menu new ui

### PR DESCRIPTION
**Pull request by elliottyates**
- extends overrides for consistency to the regular a elements
- updates .text color, font-weight and help menu background color to match new UI defaults

Here's how it looks for regular links:

screen shot 2016-03-31 at 1 50 42 pm

And for the disabled (non-linked) help menu items: (note that the light-blue highlighting is just manual text selection to show which section is not rendering as a link)

screen shot 2016-03-31 at 1 50 49 pm

### CODE REVIEW

File: css/help_options.css [hunk 1]
```diff
@@ -8,31 +8,31 @@
...
+    padding: 10px 25px;
     text-decoration: none;
 }
 
-#help-dialog #help-dialog-options li a .text,
-#help-dialog #help-dialog-options li span.info-no-url .text {
-    color:#1967aa;
+#help-dialog #help-dialog-options li a .text,  /* canvas default */
+#help-dialog #help-dialog-options li span.info-no-url .text {  /* custom */
+    color: #1e7eca;
```
Comment (sapnamysore) [inline css/help_options.css:26]: Color change? And I see the removal of bold style-  it doesn't look it was rendered as bold before as well?

**Comment by sapnamysore** [comment]
:+1:

File: css/help_options.css (hunk 1 above)
Comment (elliottyates) [inline css/help_options.css:26]: The default Canvas styling in the old UI did in fact make the `.text` bold indirectly via the parent `.a` tag.

Canvas styling:

```
#help-dialog #help-dialog-options li a {
    border-bottom: 1px solid #d6d6d6;
    display: block;
    padding: 12px;
    text-decoration: none;
    font-weight: bold;
}
```

In the new UI they removed that bold styling in the default menu, from what I can see.

Comment (elliottyates) [inline css/help_options.css:26]: Color change also matches the new UI default.

Comment (sapnamysore) [inline css/help_options.css:26]: Great! Thanks!

**Comment by elliottyates** [comment]
> [2:21 PM] neil_patch @elliott: they look good! :+1:

**Comment by elliottyates** [comment]
@sapnamysore fyi minor update pushed after your go-ahead, would you mind a quick second thumb?

File: main.css [hunk 2]
```diff
@@ -2,7 +2,5 @@
 
 @import url('css/feedback.css');
  
-@import url('css/help_dialog.css');
```
Comment (sapnamysore) [inline main.css]: Should this reference be help_options.css?

Comment (elliottyates) [inline main.css]: Actually it turns out we don't use this file at all (cf `#devops` in slack), so I updated this to reflect the merged state of develop, which has no main.css -- thanks for following-up!

**Pull request merged**